packages/bootstrap-4/test/Form.test.tsx

```diff
@@ -198,3 +198,38 @@ describe("single fields", () => {
     expect(tree).toMatchSnapshot();
   });
 });
+
+describe("Form", () => {
+  test("Renders with appropriate refs", () => {
+    function FormWithRef() {
+      const ref = useRef<
+        CoreForm<{
+          field: string;
+        }>
+      >(null);
+    
+      return (
+        <Form<{
+          field: string;
+        }>
+          ref={ref}
```

Comment:
How exactly does this snapshot ensure that the `ref` property is properly passed down? What exactly are we testing here -- just to make sure that the rendering doesn't crash? (if so, we wouldn't need a snapshot)

Also, I'm having trouble seeing why the old typings would break this test. What would happen with the old types?